src/components/Footer/FooterExtendedNavList/FooterExtendedNavList.test.tsx

```diff
@@ -0,0 +1,96 @@
+/*  eslint-disable jsx-a11y/anchor-is-valid, react/jsx-key */
+
+import React from 'react'
+import { act, fireEvent, render } from '@testing-library/react'
+
+import { FooterExtendedNavList } from './FooterExtendedNavList'
+
+const links = [
+  [
+    'Types of Cats',
+    ...Array(2).fill(
+      <a className="usa-footer__secondary-link" href="#">
+        Cheetah
+      </a>
+    ),
+  ],
+  [
+    'Musical Gifts',
+    ...Array(3).fill(
+      <a className="usa-footer__secondary-link" href="#">
+        Purple Rain
+      </a>
+    ),
+  ],
+]
+
+describe('FooterExtendedNavList component', () => {
+  it('renders without errors', () => {
+    const { container } = render(<FooterExtendedNavList nestedLinks={links} />)
+    expect(container.querySelector('ul')).toBeInTheDocument()
+  })
+
+  it('renders headings', () => {
+    const { container, getByText } = render(
+      <FooterExtendedNavList nestedLinks={links} />
+    )
+    expect(container.querySelectorAll('h4')).toHaveLength(2)
+    expect(getByText('Types of Cats')).toBeInTheDocument()
+    expect(getByText('Musical Gifts')).toBeInTheDocument()
+  })
+
+  it('renders links', () => {
+    const { container, getAllByText } = render(
+      <FooterExtendedNavList nestedLinks={links} />
+    )
+    expect(container.querySelectorAll('a')).toHaveLength(5)
+    expect(getAllByText('Purple Rain')).toHaveLength(3)
+    expect(getAllByText('Cheetah')).toHaveLength(2)
+  })
+
+  describe('isMobile', () => {
+    it.todo(
+      'renders mobile view when client window width is less than threshold'
```

Comment:
I can't figure out how to test window resize with running into typescript `read-only` limitations on reassigning `window` objects during a test.

Reply:
could you use the `resizeTo` method instead of reassigning the dimensions? https://developer.mozilla.org/en-US/docs/Web/API/Window/resizeTo

imo this would be a great test to have!

Reply:
Tried out `resizeTo` but its not implemented. jest incorporates JSDOM to override `window` - and JSDOM doesn't support all the same methods.